docs/content/guides/developer/getting-started/app-frontends.mdx

```diff
@@ -0,0 +1,171 @@
+---
+title: Connect a Frontend to a Move Package
+sidebar_label: Connect a Frontend
+description: Connect a React interface to a simple Move package to create a full stack application.
+keywords: [ first dapp, first package, move package, first move package, hello world, hello, world, react, react dapp, react ui, connect react dapp, add frontend, add ui, create fullstack app, fullstack app, full stack, full stack dapp ]
+pagination_prev: null
+pagination_next: null
+---
+
+In the previous guide, [Hello, World!](/guides/developer/getting-started/hello-world.mdx), you deployed a Move package and interacted with it to create an object that stored the text "Hello world!"
+
+This guide demonstrates how to connect a React interface to that Hello, World! package, giving any user a way to interact with the Move package from their browser and set a custom greeting.
+
+<Tabs className="tabsHeadingCentered--small">
+<TabItem value="prereq" label="Prerequisites">
+
+- [x] [Install the latest version of Sui](/guides/developer/getting-started/sui-install.mdx).
+
+- [x] [Configure the Sui client](/guides/developer/getting-started/configure-sui-client.mdx).
+
+- [x] [Create a Sui address](/guides/developer/getting-started/get-address.mdx).
+
+- [x] [Get SUI Testnet tokens](/guides/developer/getting-started/get-coins.mdx).
+
+- [x] Complete the [Hello, World!](/guides/developer/getting-started/hello-world.mdx) guide and have your published Move package's ID. 
+
+- [x] Install [`pnpm`](https://pnpm.io/installation) to use as the package manager.
+
+- [x] Create a [Slush](https://slush.app/) wallet.
+
+</TabItem>
+</Tabs>
+
+## Call the Move package 
+
+First, confirm that you have [followed the Hello, World!](https://github.com/MystenLabs/sui-stack-hello-world.git) example guide and are within the `sui-stack-hello-world/move/hello-world` directory in your CLI.
+
+Then, verify your Move package is still available on Testnet by obtaining its object information:
+
+```
+$ sui client object <PACKAGE_ID>
+```
```

Comment:
What if they are coming back to this and no longer know their package ID? sui client objects to get it, maybe?